File: SmartHouseBackend/SmartHouse.Core/Model/SmartHomeDevices/AirConditioner.cs
namespace SmartHouse.Core.Model.SmartHomeDevices
{
    public enum Mode
    {
        Cooling,
        Heating,
        Automatic,
        Ventilation
    }
    public class AirConditioner : SmartDevice
    {
        public double MinTemperature { get; set; } = 14;
        public double MaxTemperature { get; set; } = 30;
        public List<Mode> Modes { get; set; } = new();
        public Mode CurrentMode { get; set; } = Mode.Automatic;
    }
}

File: SmartHouseFrontend/src/app/shared/models/AirConditioner.ts
import { TypeOfPowerSupply } from "./PowerSupply";
export enum Mode{
    Cooling,
    Heating,
    Automatic,
    Ventilation
}
export interface RegisterAirConditionerDTO{
    name: string,
    powerSupply: TypeOfPowerSupply,
    powerUsage?:number,
    smartPropertyId: string,
    image: string,
    imageType:string,
    minTemperature:number,
    maxTemperature:number,
    modes: Mode[]
}